src/shopDetail/ShopDetail.tsx

```diff
@@ -92,8 +103,8 @@ export const ShopDetail = ({ route }: TProps) => {
               label={t('SHOP_DETAIL.REDEEM.BUTTON')}
               loading={isRedeemStatusLoading}
               onPress={() => {
-                trackSelfDescribingEvent('buttonClick', { button_name: 'redeem-cta' }, { reward: rewardTrackingData });
-                toggleRedeemModalConfirmationOpen();
+                const stickyOffset = 48 + 16; // This equals the area height of the redeem button, which equals the button height (48) + margin (16)
```

Comment:
could you put them in a const and reuse it here? So when they get changed, this also gets updated.